.travis.yml

```diff
@@ -27,9 +27,9 @@ jobs:
         - pip3 install pyyaml
         - pip3 install aura.tar.gz
       script:
-        - python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.11 --no-upstream-fetch && python3 makeBuild.py
+        - python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.13 --no-upstream-fetch && python3 makeBuild.py
```

Comment:
Should we change it to 4.14 in the validate step for 4.14 branch?
@aireilly @kalexand-rh WDYT?

Reply:
Makes sense to me!

Reply:
I agree.

Reply:
Shouldn't this change be made in the main branch too though? So they stay in sync?